docs/src/content/docs/tutorials/flutter-firebase-login.mdx

```diff
@@ -110,6 +110,36 @@ We need to follow the [firebase_auth usage instructions](https://pub.dev/package
 Remember to update the `google-services.json` on Android and the `GoogleService-Info.plist` & `Info.plist` on iOS, otherwise the application will crash.
 :::
 
+## Caching Implementation
+
+### Purpose of Caching
+
+The caching mechanism in this login flow provides several key benefits:
+
+- **Improved User Experience**: By caching the user's authentication state, we eliminate unnecessary sign-in prompts and provide seamless transitions between app sessions.
+- **Offline Support**: The cached authentication state allows the application to function even with intermittent internet connectivity.
+- **Performance Optimization**: Caching reduces authentication checks, improving app responsiveness.
+
+### Caching Details
+
+#### Caching Mechanism
+
+- Implementation using shared preferences for persistent storage
+- Secure storage of authentication tokens
+- User state management through local caching
+
+#### Caching Strategy
+
+- When cache is updated: After successful login, registration, and token refresh
+- When cache is read: App startup and authentication state checks
+- Cache invalidation rules and refresh policies
+
+#### Error Handling
+
+- Handling expired cached credentials
+- Cache corruption recovery
+- Fallback mechanisms for cache failures
```

Comment:
I don't think this fits in with the rest of the tutorial and a lot of this information doesn't apply to the cache usage in this tutorial.